test/fixtures/es-modules/package-cjs-named-error/fail.cjs

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  comeOn: 'fhqwhgads'
+  comeOn: 'fhqwhgads',
+  rightNow: 'abcde',
```

Comment:
```suggestion
  everybody: 'to the limit',
```

Sticking with the homestar theme... this can be ignored 😇

Reply:
TIL… 😂